wordpress-mobile/WordPress-iOS
# Fix dirty repository after internal build

https://github.com/wordpress-mobile/WordPress-iOS/pull/12454 added a baseline set of internal icons, in order to get rid of some missing images warnings when checking out `develop` for the first time. 

As a small side effect, after every `internal` build the repository is dirty, because the icons are modified and the entry in `.gitignore` doesn't work anymore because the files are now tracked. 

In order to fix this, I tinkered a bit with `git update-index --assume-unchanged`, but it works only as a local setting and it can cause some weird conflicts that can cause confusion to developers that don't have context about this. 

So, this PR proposes a different approach, removing the assets from the repository and creating the baseline set in an additional step of `rake dependencies` that runs only if the icons are missing. 

### To test:
1. Checkout this branch.
2. Run `rake dependencies`.
3. Open XCode and build the `internal` scheme.
4. Verify that there are no warnings related to missing icons.
5. Verify that the repository is clean after the build.

PR submission checklist:
